### DISCUSSION TITLE
The XMonad Logo Contest: Voting

### DISCUSSION THREAD

**Discussion by slotThe** [xmonad/xmonad#359] [Announcements]
# We have a winner 

[A winner has been chosen](https://github.com/xmonad/xmonad/discussions/359#discussioncomment-1977016): 

| Name                                                                                                                        | Icon Size                                                                                                                            | Logo Size                                                                                                                            |
|:----------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------------:|
| @literallyunplayable's [minimalist](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1616411)             | <img src="https://user-images.githubusercontent.com/50166980/147848868-15c011db-8f00-47bd-a7e3-a98b01e196dc.png" width=16> | <img src="https://user-images.githubusercontent.com/50166980/147848868-15c011db-8f00-47bd-a7e3-a98b01e196dc.png" width=50> |

---

# Original OP

This is the voting part for the official XMonad logo contest!

### Voting (2022-01-01–2022-01-15)

For voting, we will use an [instant-runoff] type system.  In short:

  - Users rank __a minimum of five (5) submissions__ (by name) according to their preferences in decreasing order (you can vote for your own submission).
  - If a logo wins a majority of first-preference votes (> 50%) it is chosen.
  - If not, we eliminate the logo with the least amount of first preference votes, exposing the second preference of those who voted for it.
  - Rinse and repeat until we have a majority.
  - If at the end of all that we still don't have a majority, we will act as a tiebreaker and internally vote for a winner from the remaining logos.

We will retain veto power in case of inappropriate logos.  Team members whose submitted entries are amongst the remaining choices will not partake in the potential tiebreaking vote.

__Important Note:__ We have chosen a single entry for people submitting multiple logos that only differ in their colour scheme.  Please vote for the logo itself and not for the concrete colour scheme, which may or may not be subject to change!

__Important Note²:__ As noted by @literallyunplayable [below](https://github.com/xmonad/xmonad/discussions/359#discussioncomment-1894151):  

> Many of the icon sized images look quite fuzzy. It is possible to carefully align the artwork with a pixel grid in inkscape to produce sharper icon sized images of the winner. Keep this in mind before discounting a submission for having a difficult to read 24px icon.

[instant-runoff]: https://en.wikipedia.org/wiki/Instant-runoff_voting

### Submissions (in random order)

| Name                                                                                                                        | Icon Size                                                                                                                            | Logo Size                                                                                                                            |
|:----------------------------------------------------------------------------------------------------------------------------|:------------------------------------------------------------------------------------------------------------------------------------:|:------------------------------------------------------------------------------------------------------------------------------------:|
| @la-ninpre's [lambda-x](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1615106)                         | <img src="https://user-images.githubusercontent.com/57688959/140994704-70641703-a0f8-450d-9205-c1d38fd03937.png" width="16">         | <img src="https://user-images.githubusercontent.com/57688959/140994704-70641703-a0f8-450d-9205-c1d38fd03937.png" width="50">         |
| @leohanyan's [vmon](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1691909)                             | <img src="https://user-images.githubusercontent.com/73842241/143188992-bdab796f-63f2-4175-93eb-1aa3a22960fc.png" width="16">         | <img src="https://user-images.githubusercontent.com/73842241/143189026-1f279428-c9d6-4016-97c5-ff014a434495.png" width="50">         |
| @byte-wolf's [Crowbar](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1632886)                          | <img src="https://user-images.githubusercontent.com/85117932/141499859-e8d74cf8-d2f3-4257-8099-81fff21cfbcd.png" width="16">         | <img src="https://user-images.githubusercontent.com/85117932/141499867-2779a1f5-8849-4910-b83f-25ea992de600.png" width="50">         |
| @IrinaBalabay's [Сonstructor](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1717553)                   | <img src="https://user-images.githubusercontent.com/95201612/143919157-16e3c3dc-7d8b-48db-8fed-aafc0c300065.png" width="16">         | <img src="https://user-images.githubusercontent.com/95201612/143919150-a95398ca-92a4-48dd-a315-e58dbf9f694a.png" width="50">         |
| @weiss-d's [Minimal XM 2](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1823187)                       | <img src="https://user-images.githubusercontent.com/62437174/146378287-2c39f26f-0af1-4e78-bb44-b7618a2d634f.png" width="16">         | <img src="https://user-images.githubusercontent.com/62437174/146378337-9e3a31bd-28b9-4a40-b4e3-c01e64b28110.png" width="50">         |
| @astynax's [XClock](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1621738)                             | <img src="https://user-images.githubusercontent.com/4694635/141174552-dd51e992-af35-4371-85ed-5b8973b80e87.png" width="16">          | <img src="https://user-images.githubusercontent.com/4694635/141174552-dd51e992-af35-4371-85ed-5b8973b80e87.png" width="50">          |
| @muelphil's [curvy wide](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1621306)                        | <img src="https://user-images.githubusercontent.com/31062727/142772121-7f3a0fbb-7116-429b-82fa-c3bf38a0a4ef.png" width="16">         | <img src="https://user-images.githubusercontent.com/31062727/142772121-7f3a0fbb-7116-429b-82fa-c3bf38a0a4ef.png" width="50">         |
| @akze-rulez's [Homenad](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1624667)                         | <img src="https://user-images.githubusercontent.com/94106895/141276740-ef7b6f0a-5ab0-4269-8717-a214d3a22a7e.png" width="16">         | <img src="https://user-images.githubusercontent.com/94106895/141276808-29324e93-6ed1-4e56-ba8c-abef06b3cfb3.png" width="50">         |
| @byte-wolf's [Gradient Patriot](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1632924)                 | <img src="https://user-images.githubusercontent.com/85117932/141501029-7bf514be-32b0-41b9-809a-ead2f47c0969.png" width="16">         | <img src="https://user-images.githubusercontent.com/85117932/141501035-b42ade31-b94d-4600-a8eb-4799f0a908b3.png" width="50">         |
| @leohanyan's [monado](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1678597)                           | <img src="https://user-images.githubusercontent.com/73842241/142776598-5579cf33-b77b-4290-b3f8-21de398d9260.png" width="16">         | <img src="https://user-images.githubusercontent.com/73842241/142776600-d888cdb7-a626-4411-8a39-260d2c6277a1.png" width="50">         |
| @ssfwshutterbug's [ssfwshutterbug's submission](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1626495) | <img src="https://user-images.githubusercontent.com/85094154/141320330-c48841bb-f790-4433-8540-af119e8a913b.png" width="16">         | <img src="https://user-images.githubusercontent.com/85094154/141320330-c48841bb-f790-4433-8540-af119e8a913b.png" width="50">         |
| @emptyflask's [X >>=](https://user-images.githubusercontent.com/28287/141144231-d814f004-0113-4e33-a326-1dc7d01078ab.png)   | <img src="https://user-images.githubusercontent.com/28287/141144231-d814f004-0113-4e33-a326-1dc7d01078ab.png" width=16>              | <img src="https://user-images.githubusercontent.com/28287/141144211-d010d9fe-604a-430f-b9f0-03f4d6f91287.png" width=50>              |
| @joelchrono12's [Wide X v1](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1615622)                     | <img src="https://user-images.githubusercontent.com/18448587/141011569-25b3fde2-adf3-4a8a-afee-51d57f4c4864.png" width="20">         | <img src="https://user-images.githubusercontent.com/18448587/141011569-25b3fde2-adf3-4a8a-afee-51d57f4c4864.png" width="50">         |
| @namanUIUC's [XMonad Spaces](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1622725)                    | <img src="https://user-images.githubusercontent.com/32119068/141201917-ca02a301-e171-4f9f-947b-6758738cf527.png" width="16">         | <img src="https://user-images.githubusercontent.com/32119068/141202089-d433a404-e1fc-4e17-a9f8-a7e1470ecfb0.png" width="50">         |
| @byte-wolf's [Phoenix](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1632837)                          | <img src="https://user-images.githubusercontent.com/85117932/141495119-22b2bc2a-c6db-4aa3-a78f-c8f82698520d.png" width="16">         | <img src="https://user-images.githubusercontent.com/85117932/141495111-35ef2f9e-58bb-416a-8a55-c8f3bb1e6187.png" width="50">         |
| @greenfierydragon's [greenmonad](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1624152)                | <img src="https://user-images.githubusercontent.com/21034889/141256506-3e715dd4-8d12-4e21-83a6-99e3f609e740.png" width="16">         | <img src="https://user-images.githubusercontent.com/21034889/141256449-187df7a5-7f49-48b3-b7cf-8c0d8af5dbe7.png" width="50">         |
| @muelphil's [muelphil's submission 2](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1621516)           | <img src="https://user-images.githubusercontent.com/31062727/145732104-583861ab-f187-4347-ac63-4e534772bc6a.png" width="16">         | <img src="https://user-images.githubusercontent.com/31062727/145732104-583861ab-f187-4347-ac63-4e534772bc6a.png" width="50">         |
| @astynax's [XMondrian](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1638651)                          | <img src="https://user-images.githubusercontent.com/4694635/147133701-c92b4a0b-4ee4-4d4d-9ed9-d357352f45b1.png" width="16">          | <img src="https://user-images.githubusercontent.com/4694635/147133704-510b4c35-ac1f-404b-874f-37220ced1590.png" width="50">          |
| @slotThe's [Haskell meets X](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1607598)                    | <img src="https://user-images.githubusercontent.com/50166980/141111800-f8916dd2-6965-45f1-90be-aa71b46e7eb0.png" width="16">         | <img src="https://user-images.githubusercontent.com/50166980/141111800-f8916dd2-6965-45f1-90be-aa71b46e7eb0.png" width="50">         |
| @array-in-a-matrix's [New Monad](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1622002)                | <img src="https://user-images.githubusercontent.com/78233840/141186808-c39c9446-994f-486d-8989-5392d776b6ab.png" width="16">         | <img src="https://user-images.githubusercontent.com/78233840/141186440-141c1337-6d7e-4f20-998a-9dcd36170791.png" width="50">         |
| @azraeljd's [>X=](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1722065)                               | <img src="https://user-images.githubusercontent.com/94840719/144046700-d26a3791-eed2-4fe4-8792-8d9d97ab2dbb.png" width="16">         | <img src="https://user-images.githubusercontent.com/94840719/144046762-c0d271e4-7d61-41fe-afda-0d99a8d0afb5.png" width="50">         |
| @bugreporteur's [bugreporteur's submission](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1615943)     | <img src="https://user-images.githubusercontent.com/94020145/141015368-f4dbe710-0d71-4d65-ae8c-eb09bcefdb04.png" width="16">         | <img src="https://user-images.githubusercontent.com/94020145/141015368-f4dbe710-0d71-4d65-ae8c-eb09bcefdb04.png" width="50">         |
| @fornons's [4screens](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1649642)                           | <img src="https://user-images.githubusercontent.com/92436212/141946203-10606ec4-26d0-4cf2-a798-331626ee544a.png" width="16">         | <img src="https://user-images.githubusercontent.com/92436212/141946236-99b9e8a3-82b3-496f-9c29-1a624546eb6b.png" width="50">         |
| @MatthiasBenaets's [Modern Basic 2](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1618697)             | <img src="https://user-images.githubusercontent.com/89214559/141098370-cc95bcc5-edfc-4440-8514-6e32656d29fe.png" width="16">         | <img src="https://user-images.githubusercontent.com/89214559/141098370-cc95bcc5-edfc-4440-8514-6e32656d29fe.png" width="50">         |
| @MatthiasBenaets's [Modern Basic 1](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1618102)             | <img src="https://user-images.githubusercontent.com/89214559/141347801-ac86056a-f5bf-41e0-8b1b-0a5ffe751ab1.png" width="16">         | <img src="https://user-images.githubusercontent.com/89214559/141347801-ac86056a-f5bf-41e0-8b1b-0a5ffe751ab1.png" width="50">         |
| @leohanyan's [powermonad](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1702616)                       | <img src="https://user-images.githubusercontent.com/73842241/143500670-677bbcb4-87ae-472d-923a-c7cce07c82ec.png" width="16">         | <img src="https://user-images.githubusercontent.com/73842241/143500675-e72afd0f-c699-4cd9-85fa-7a22cd5027e1.png" width="50">         |
| @weiss-d's [Minimal XM 1](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1823187)                       | <img src="https://user-images.githubusercontent.com/62437174/146378272-eae05fcd-afb9-44f6-9302-0cdf950a97cc.png" width="16">         | <img src="https://user-images.githubusercontent.com/62437174/146378316-14d009b2-577b-43d9-890c-88e2de69c6c7.png" width="50">         |
| @Napeer's [XPinWheel](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1615887)                           | <img src="https://raw.githubusercontent.com/Napeer/xmonad-logo/main/xmonadlogo-tiny.png" width="16">                                 | <img src="https://raw.githubusercontent.com/Napeer/xmonad-logo/main/xmonadlogo-less-tiny.png" width="50">                            |
| @reinielfc's [Xemonad](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1708303)                          | <img src="https://user-images.githubusercontent.com/66280594/143667227-01be7da1-36e0-4484-834f-c62c4dd4d942.png" width="16">         | <img src="https://user-images.githubusercontent.com/66280594/143667233-a2d86e50-d50d-4535-b345-b929bd516291.png" width="50">         |
| @literallyunplayable's [minimalist](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1616411)             | <img src="https://user-images.githubusercontent.com/50166980/147848868-15c011db-8f00-47bd-a7e3-a98b01e196dc.png" width=16> | <img src="https://user-images.githubusercontent.com/50166980/147848868-15c011db-8f00-47bd-a7e3-a98b01e196dc.png" width=50> |
| @SocialDiablo's [Xmonovad](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1638543)                      | <img src="https://user-images.githubusercontent.com/88990609/141683539-28a2ffa3-dc95-457b-8e9b-749d2762b62e.png" width="16">         | <img src="https://user-images.githubusercontent.com/88990609/141683544-22810332-61fe-4e71-a12d-9af158e5c122.png" width="50">         |
| @MikeValo's [racoon](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1618259) | <img src="https://user-images.githubusercontent.com/94018613/147887868-3b24108c-cb49-404d-917c-510970c96d22.png" width="16"> | <img src="https://user-images.githubusercontent.com/94018613/147887868-3b24108c-cb49-404d-917c-510970c96d22.png" width="50"> |

### Example vote

As an example, consider the following vote (simply going down the list):

1.  lambda-x
2.  vmon
3.  Crowbar
4.  Сonstructor
5.  Minimal XM 2
6.  XClock
7.  curvy wide
8.  Homenad
9.  Gradient Patriot
10. monado
11. ssfwshutterbug's submission
12. X >>=

### Ways to Vote

There are several ways in which you can vote, depending on your preference.

1. Reply to this thread!
2. Reply to the relevant [thread on the mailing list](https://mail.haskell.org/pipermail/xmonad/2022-January/015494.html)

**Comment by la-ninpre** [comment]
1. lambda-x
2. minimalist
3. gradient patriot
4. curvy wide
5. monado
6. xemonad
7. submission 2

**Reply by slotThe** [reply]
They can!  Sometimes it is even required that people rank their own submission (or themselves, if applicable) as number one but we chose to ignore that for this contest

**Reply by la-ninpre** [reply]
okay then, thanks!

**Comment by geekosaur** [comment]
1. Haskell meets X
2. New Monad
3. XPinWheel
4. powermonad
5. minimalist

**Comment by NiHaiden** [comment]
1. Phoenix
2. Homenad
3. Gradient Patriot
4. Crowbar
5. Modern Basic 1

**Comment by emptyflask** [comment]
1. Phoenix
2. curvy wide
3. crowbar
4. Haskell meets X
5. minimalist
6. Xemonad
7. Gradient Patriot
8. X >>=

**Comment by literallyunplayable** [comment]
1. minimalist
2. phoenix
3. modern basic 2
4. lambda x
5. minimal xm 1
6. monado
7. minimal xm 2
8. gradient patriot

**Reply by literallyunplayable** [reply]
Many of the icon sized images look quite fuzzy. It is possible to carefully align the artwork with a pixel grid in inkscape to produce sharper icon sized images of the winner. Keep this in mind before discounting a submission for having a difficult to read  24px icon.

**Comment by bubblegumbubble** [comment]
1. minimalist
2. Сonstructor
3. Haskell meets X
4. bugreporteur's submission
5. muelphil's submission 2

**Comment by ValoMichal** [comment]
1 racoon (ofc)
2 minimalist
3 xemonad
4 minimal xm 2
5 crowbar

**Reply by slotThe** [reply]
Oh, I didn't realise you added a license!  Totally my fault, sorry about that!  It's a bit awkward but if you tell me which of the five designs you'd like to submit I can edit the OP

**Reply by ValoMichal** [reply]
oh really? i tought its too stupid to be in voting phase lol
i dont want to inconvinience you but we can add this one for fun
![xmonadA - Copy (2)](https://user-images.githubusercontent.com/94018613/147887868-3b24108c-cb49-404d-917c-510970c96d22.png)
 for name u can use racoon or whatever

**Comment by UlisesAlexanderAM** [comment]
1. Haskell meets X
2. minimalist
3. XMondrian
4. Phoenix
5. X >>=

**Comment by mgsloan** [comment]
1. curvy wide
2. Phoenix
3. XMondrian
4. minimalist
5. racoon

**Reply by slotThe** [reply]
@mgsloan it would be neat if you could rank two more submissions (we settled on a minimum of five so votes don't go into the æther in the early rounds)

**Reply by mgsloan** [reply]
Updated!

**Comment by psibi** [comment]
1. Gradient Patriot 
2. Pheonix
3. Haskell meets X
4. minimalist
5. Xmonovad

**Comment by byte-wolf** [comment]
1. Phoenix
2. Monado
3. curvy wide
4. minimalist
5. Gradient Patriot
6. Haskell meets X
7. lambda-x

**Comment by kronn** [comment]
1. minimalist
2. Haskell meets X
3. Modern Basic 2
4. Phoenix
5. Crowbar
6. Minimal XM 2
7. X >>=
8. monado
9. muelphils submission 2
10. racoon

**Comment by 17bell11** [comment]
1. Phoenix
2. curvy wide
3. Haskell meets X
4. minimalist
5. Crowbar

**Comment by D3vZro** [comment]
1. Haskell meets X
2. Crowbar
3. Phoenix
4. minimalist
5. Monado

**Comment by muelphil** [comment]
1. curvy wide
2. Modern Basic 2
3. Minimal XM 1
4. Minimal XM 2
5. vmon
6. X >>=
7. Haskell meets X
8. XMonad Spaces

**Comment by sytek1** [comment]
1. curvy wide
2.  minimalist
3.  Phoenix
4. Haskell meets X
5.  Gradient Patriot

**Reply by geekosaur** [reply]
May I remind people that they are to vote for at least 5 entries?

**Comment by astynax** [comment]
1. curvy wide
1. lambda-x
1. Phoenix
1. minimalist
1. XMonad Spaces

**Comment by array-in-a-matrix** [comment]
1. New Monad
2. Minimal XM 2
3. Phoenix
4. Crowbar
5. Gradient Patriot

**Comment by 0x08088405** [comment]
1. minimalist
2. curvy wide
3. phoenix
4. muelphil's submission 2
5. haskell meets x
6. new monad
7. lambda-x
8. crowbar
9. gradient patriot
10. xclock 🥺

**Comment by TheMC47** [comment]
1. minimalist
2. Modern Basic 2
3. Haskell meets X
4. Crowbar
5. curvy wide
6. lambda-x

**Comment by aij** [comment]
1. curvy wide
1. Crowbar
1. XMondrian
1. Haskell meets X
1. bugreporteur's submission
1. Phoenix
1. lambda-x

**Comment by agnantis** [comment]
1. minimalist
2. Minimal XM 2
3. curvy wide
4. Gradient Patriot
5. muelphil's submission 2

**Comment by slotThe** [comment]
1. curvy wide
2. minimalist
3. Haskell meets X
4. Phoenix
5. Minimal XM 1
6. Gradient Patriot
7. Minimal XM 2
8. Crowbar

**Comment by liskin** [comment]
1. minimalist
2. Modern Basic 2
3. Minimal XM 1
4. Phoenix
5. Crowbar
6. Modern Basic 1

**Comment by noaccOS** [comment]
1. minimalist
2. curvy wide
3. Gradient Patriot
4. Pohenix
5. monado

**Comment by wineee** [comment]
1.Haskell meets X
2.Phoenix
3.monado
4.Gradient Patriot
5.powermonad

**Comment by leohanyan** [comment]
1. Monado
2. Phoenix
3. Minimalist
4. Crowbar
5. Powermonad

**Comment by kolmodin** [comment]
1. minimalist
2. Crowbar
3. muelphil's submission 2
4. Haskell meets X
5. Gradient Patriot

**Comment by adokce** [comment]
1. curvy wide
2. muelphil's submission 2
3. Xmonovad
4. Haskell meets X
5. vmon

**Comment by gvolpe** [comment]


**Reply by slotThe** [reply]
@gvolpe As per the rules in the OP please vote for at least five candidates

**Comment by wejn** [comment]
1. Crowbar
2. curvy wide
3. Phoenix
4. X >>=
5. Xemonad
6. minimalist
7. Gradient Patriot

**Comment by 11xx** [comment]
1. minimalist
2. Haskell meets X
3. Modern Basic 2
4. Modern Basic 1
5. Gradient Patriot

**Comment by Kritiqual** [comment]
1. Pheonix
2. Curvy wide
3. Haskell meets x
4. Modern basic 1
5. \>X=

**Comment by brunefig** [comment]
1. minimalist
2. curvy wide [modification proposed by tim-kilian](https://github.com/xmonad/xmonad/discussions/359#discussioncomment-1935092)
3. Haskell meets X
4. Gradient Patriot
5. XMondrian
6. New Monad
7. \>X=

**Comment by stridervc** [comment]
1. curvy wide                                       
2. Phoenix                                          
3. Gradient Patriot                                 
4. Modern Basic 1                            
5. 4screens

**Comment by herecy** [comment]
1. Phoenix
2. XMonad Spaces
3. Haskell meets X
4. minimalist
5. curvy wide

**Comment by nomeata** [comment]
1. minimalist
2. Haskell meets X
3. Phoenix
4. Gradient Patriot
5. New Monad

**Comment by eshrh** [comment]
1. minimalist
2. Phoenix
3. Haskell meets X
4. curvy wide
5. Gradient Patriot

**Comment by CantoroMC** [comment]
1. curvy wide
2. monado
3. Haskell meets X
4. `>X=`
5. Modern Basic 1

**Comment by superdamo** [comment]
1. minimalist
2. Haskell meets X
3. Phoenix
4. Gradient patriot
5. Crowbar

**Comment by l29ah** [comment]
haskell meets x
homenad
gradient patriot
minimalist
constructor
new monad
phoenix

**Comment by YakBarber** [comment]
1. Haskell meets X
2. minimalist
3. X >>=
4. crowbar
5. Gradient Patriot
6. Phoenix

**Comment by kelvin-273** [comment]
1 Minimalist
2 Minimal XM2
3 Pheonix
4 Curvy meets X
5 Minimal XM1
6 Haskell Meets X

**Comment by kcrunnels1515** [comment]
1. curvey wide
2. Modern Basic 1
3. powermonad
4. X >>=
5. Crowbar
6. Haskell Meets X

**Comment by sangster** [comment]
There are a lot of great submissions!

1. @slotThe's Haskell meets X
2. @array-in-a-matrix's New Monad
3. @muelphil's curvy wide
4. @byte-wolf's Phoenix
5. @muelphil's muelphil's submission 2
6. @MatthiasBenaets's Modern Basic 1
7. @byte-wolf's Crowbar

**Comment by thedward** [comment]
1. Phoenix
2. Minimal XM 1
3. New Monad
4. Crowbar
5. Minimal XM 2
6. muelphil's submission 2

**Comment by Aditeya** [comment]
1. minimalist
2. New Monad
3. Gradient Patriot
4. Haskell meets X
5. XClock
6. vmon
7. 4screens
8. Modern Basic 1

**Comment by oneself** [comment]
1. Modern Basic 1
2. minimalist
3. Haskell meets X
4. 4screens

**Reply by slotThe** [reply]
@oneself As per the rules in the OP please vote for at least five candidates

**Comment by texttheater** [comment]
1. 4 screens
2. Modern Basic 2 
3. Constructor
4. curvy wide
5. Homenad

**Comment by eugene-reim** [comment]
1. Phoenix
2. Gradient Patriot
3. minimalist
4. curvy wide
5. Minimal XM 1
6. vmon
7. lambda-x 
8. monado
9. Minimal XM 2

**Comment by Vvamp** [comment]
1. Minimal XM 2
2. Lambda X
3. Curvy wide
4. Minimalist
5. Gradient Patriot

**Comment by skowriii** [comment]
1. Monado
2. 4screens
3. Crowbar
4. Phoenix
5. minimalist

**Comment by greenbourne277** [comment]
1. XMondrian
2. monado
3. minimalist
4. Minimal XM 2
5. Modern Basic 2

**Comment by CyanideForBreakfast** [comment]
1. minimalist
2. Haskell meets X
3. New Monad
4. X >>=
5. Modern Basic 1

**Comment by emiham** [comment]
1. minimalist
2. curvy wide
3. Phoenix
4. Gradient Patriot
5. Haskell meets X

**Comment by tim-kilian** [comment]
1. curvy wide (and maybe not wide with just the X)
![xmonad-logo](https://user-images.githubusercontent.com/5788010/148683473-47aeea61-bbcf-4057-8768-2c26edb1fc07.png)
2. Minimal XM (idea is great! But color coding not perfect)
3. minimalist (maybe with different color code)
4. Сonstructor
5. Haskell meets X
6. Phoenix
7. lambda-x

**Reply by gvolpe** [reply]
I can get behind your proposals for (1) and (3) 👍🏽

**Reply by eugene-reim** [reply]
Actually your version of curvy wide is better than original. Love it!

**Reply by muelphil** [reply]
Nice job, looks clean!
Personally wouldnt have had the idea tho, as I think a logo should be more than one character (personal opinion). Making it too minimalistic might be clean but also without character. As you might see from my votes I like ideas that actually tell something about what it does or are very unique. I had >alot< of other logo ideas based on window tiling, that looked really great on paper but aweful when I traced it. God knows why. I prefer the other logos over the >>X= based logos.

**Comment by jerri** [comment]
1. Phoenix
2. Haskell meets X
3. minimalist

**Reply by geekosaur** [reply]
Please select at least 5 entries, as per the rules above.

**Comment by gvolpe** [comment]
1. curvy wide
2. minimalist
3. phoenix
4. crowbar
5. gradient patriot

**Comment by progo** [comment]
1. Minimalist                                                                                                                                              
2. Homenad                                                                                                                                                 
3. Curvy Wide                                                                                                                                              
4. Phoenix                                                                                                                                                 
5. Modern Basic 2

**Comment by EricGebhart** [comment]
1.curvy wide
2. Crowbar
3. Haskell meets X
4. Xmonad spaces
5. Modern Basic

**Reply by geekosaur** [reply]
Could you please add a fifth choice, per the instructions at the top (minimum *5* entries)?

**Reply by EricGebhart** [reply]
5. Modern basic

**Comment by eyenx** [comment]
1. Haskell meets X
2. minimalist
3. New Monad
4. Modern Basic 1
5. Crowbar

**Comment by permelin** [comment]
1. minimalist
2. curvy wide
3. Gradient Patriot
4. Phoenix
5. X >>=
6. Haskell meets X
7. Minimal XM 2
8. Minimal XM 1

**Comment by amir** [comment]
1. curvy wide
2. Gradient Patriot
3. Phoenix
4. minimalist
5. Modern Basic 1

**Reply by geekosaur** [reply]
Please choose at least 4 more logos, per the instructions at the top of this post. Instant runoff won't work very well if you only choose one and it gets eliminated.

**Reply by l29ah** [reply]
Instant runoff just sucks // https://en.wikipedia.org/wiki/Condorcet_method#Comparison_with_instant_runoff_and_first-past-the-post_(plurality)

**Reply by TheMC47** [reply]
@l29ah  thanks for the feedback, it's sadly a couple of months late :)

**Reply by geekosaur** [reply]
I'll admit I was surprised we didn't use Condorcet, but everyone else seemed happy with instant runoff so I didn't press it. And it's just a logo, not e.g. a governance decision.

**Comment by edunfelt** [comment]
1. curvy wide
2. Phoenix
3. vmon
4. XMondrian
5. Crowbar
6. Modern Basic 2
7. Constructor
8. minimalist

**Comment by aburtsev** [comment]
1. Haskell meets X
2. Modern Basic 2
3. X >>=
4. Gradient Patriot
5. minimalist

**Comment by turboMaCk** [comment]
1. Haskell meets X
2. minimalist
3. Phoenix
4. Gradient Patriot
5. New Monad

**Reply by geekosaur** [reply]
Please provide at least 5 votes, per the guidelines at the top of the page.

**Comment by davama** [comment]
1. minimalist
2. Haskell meets X
3. X >>=
4. curvy wide
5. XClock

**Comment by lukasjuhrich** [comment]
1. Haskell meets X
2. Phoenix
3. minimalist
4. bugreporteur's submission
5. curvy wide

**Comment by stepnem** [comment]
1. XClock
2. minimalist
3. lambda-x
4. Constructor
5. muelphil's submission 2
6. racoon

**Comment by vrs** [comment]
1. Phoenix
2. Gradient Patriot
3. minimalist
4. curvy wide
5. Haskell meets X

**Comment by idmitrymolchanov** [comment]
1. lambda-x
2. Modern Basic 1
3. powermonad
4. Haskell meets X
5. XClock

**Comment by orsll** [comment]
1. minimalist
2. Haskell meets X
3. curvy wide
4. Phoenix
5. Gradient Patriot
6. lambda-x
7. Crowbar

**Comment by dschrempf** [comment]
1. monado
2. Haskell meets X
3. minimalist
4. curvy wide
5. Modern Basic 2
6. Modern Basic 1

**Comment by mdemaredeb** [comment]
1. Gradient Patriot
2. minimalist
3. Crowbar
4. X >>=
5. curvy wide

**Comment by janbina** [comment]
1. Minimal XM 2
2. Minimal XM 1
3. X >>=
4. minimalist
5. Сonstructor

**Comment by akkerman** [comment]
1. Haskell meets X
2. Crowbar
3. Minimalist
4. curvy wide
5. Phoenix

**Comment by judeka** [comment]
1. minimalist
2. curvy wide
3. monado
4. Minimal XM 2
5. Constructor

**Comment by int-index** [comment]
1. curvy wide
2. minimalist
3. XClock
4. Haskell meets X
5. Сonstructor

**Comment by ken-scarf** [comment]
1. muelphil's submission 2
2. curvy wide
3. minimalist
3. 4screens

**Reply by geekosaur** [reply]
Can you vote for a fifth, per the guidelines at the top of the page?

**Comment by ento** [comment]
1. racoon
2. Xemonad 
3. curvy wide 
3. Phoenix
5. XMonad Spaces

**Comment by Pitometsu** [comment]
1. X >>=
2. curvy wide
3. Haskell meets X
4. minimalist
5. Сonstructor
6. Minimal XM 1
7. Minimal XM 2

**Reply by geekosaur** [reply]
You need to vote for 5 or more logos, per the guidelines at the top of the page.

**Comment by Filipilon** [comment]
What about "If you want to participate, please submit at most three logos (in SVG format) to this GitHub discussion by the **31st of January**." @ your page [https://xmonad.org/news/2021/10/27/xmonad-0-17-0.html](url), huh? The date is 31st of January, today is just 15th.

**Reply by slotThe** [reply]
The deadline for submissions was moved [here](https://github.com/xmonad/xmonad/discussions/343#discussioncomment-1632965); the actual discussion was edited, but the blogpost was not

**Comment by ValoMichal** [comment]
when can we expect the anouncment? (this evening, tomorow morning or later)

**Reply by slotThe** [reply]
> when can we expect the anouncment? (this evening, tomorow morning or later)

Today is the last day of voting, so you can expect a result some time tomorrow

**Reply by ValoMichal** [reply]
thx

**Comment by hw0lff** [comment]
1. minimalist
2. Crowbar
3. New Monad
4. Haskell meets X
5. xemonad
6. `>X=`
7. Gradient Patriot
8. curvy wide
9. muelphil's submission 2
10. vmon
11. Phoenix
12. Modern Basic 2

**Comment by slotThe** [comment]
And it's a wrap!  Thanks so much to everyone who voted (or submitted logos)—I don't think anyone of us could have imagined that this would get as much attention as it did!

*and the winner is..........* __minimalist__ by @literallyunplayable; congratulations!  More detailed logs with all the rounds are available [here](https://github.com/slotThe/instant-runoff/blob/master/result.txt).  That repo also contains the implementation used (in case you want to double check), as well as the input (in case you want to visualise it)!

**Reply by ValoMichal** [reply]
no way, i survived till round 11!

**Reply by TheMC47** [reply]
@MikeValo 
![image](https://user-images.githubusercontent.com/23745185/149656110-d9f6c6fe-deba-4c54-9d72-4924b77f3eff.png)

**Comment by WitherCubes** [comment]
Lock this conversation because the voting is over. I will delete this message after the conversation is locked.
